articles/quickstart/spa/vuejs/_includes/_centralized_login.md

```diff
@@ -2,475 +2,441 @@
+    );
+  }
+});
 
 Vue.config.productionTip = false;
 
 new Vue({
   router,
   render: h => h(App)
 }).$mount("#app");
-
 ```
 
-### Provide a Login Control
-
-Provide a component with controls for the user to log in and log out.
-
-::: note
-This example was created from a [Vue CLI](https://cli.vuejs.org/) template and uses Single-File Components.
-:::
-
-${snippet(meta.snippets.use)}
-
-The `@click` events on the **Log In** and **Log Out** buttons make the appropriate calls to the `AuthService` to allow the user to log in and log out. Notice that these buttons are conditionally hidden and shown depending on whether or not the user is currently authenticated.
-
-Also notice the use of `this.$auth` to access the `AuthService` instance. Login events can be handled by providing a `handleLoginEvent` method on any component.
-
-When the **Log In** button is clicked, the user will be redirected to login page.
-
-<%= include('../../_includes/_hosted_login_customization') %>
-
-When the application first starts up, a call to `renewTokens` is made that tries to reinitialize the user's login session, if it is detected that they should already be logged in. This would be the case, for example, if the user logged in and then refreshed the browser window.
+Notice that the configuration file created earlier has been imported and used to initialize the plugin. A custom redirect callback has also been supplied, which redirects the user to a protected route after they have authenticated (this is covered a bit later).
 
-### Add a Callback Component
+## Log in to the App
 
-Using Universal Login means that users are taken away from your application to a login page hosted by Auth0. After they successfully authenticate, they are returned to your application where a client-side session is set for them.
+In this section, you will provide a way for the user to log in or log out, depending on their authentication status.
 
-<%= include('../../_includes/_callback_component') %>
-
-When a user authenticates at the login page and is then redirected back to your application, their authentication information will be contained in a URL hash fragment. The `handleAuthentication` method in the `AuthService` is responsible for processing the hash.
-
-Install `vue-router` to allow callbacks to be routed properly to the `Callback` component:
-
-```bash
-npm install vue-router
-```
-
-Create a component named `Callback` and populate it with a loading indicator. The component should also call `handleAuthentication` from the `AuthService`.
-
-```js
-<!-- src/components/Callback.vue -->
+Open the `views/Home.vue` file and modify the template to include two buttons that enable the user to log in and log out:
 
+```html
 <template>
-  <div>
-    <p>Loading...</p>
+  <div class="home">
+    <img alt="Vue logo" src="../assets/logo.png" />
+    <HelloWorld msg="Welcome to Your Vue.js App" />
+
+    <!-- Check that the SDK client is not currently loading before accessing is methods -->
```

Comment:
I didn't realize this was the only section added at first ... easy to fix but might be better guidance we can provide?

Reply:
I wish there was a solution to this.

One thing I tried was using the `diff` type to highlight changes, which gives us this:

<img width="896" alt="image" src="https://user-images.githubusercontent.com/766403/65439197-b2b37000-de1e-11e9-94ca-c6c3ffce1981.png">

But it's still not perfect. In an ideal world, the new additions would be colorized normally as per the language syntax, and everything else would be greyed out.

Reply:
Something like this:

<img width="896" alt="image" src="https://user-images.githubusercontent.com/766403/65439793-a4b21f00-de1f-11e9-9a42-ea89d1b41150.png">

Reply:
I would **LOVE** to have that option. The upcoming ID labs would really use that as well. Not sure if that's possible with the code highlighting library we're using. Maybe something that @davidpatrick could help with?

Reply:
The above screenshot ☝️ is easy to do with a bit of styling. Nothing else in the docs project uses the `diff` language type so it would be easy to target.

What's not so easy is having the diff show up in normal syntax colours and grey out everything else. Looking around, there's a few SO questions and blog posts about a similar thing but nothing concrete.

I'm going to put aside some time later to try and work out a possible solution.

Reply:
We can put this aside for now since it should not block this PR. Let me know if I can help at all.